server/routes/referrals/referralsController.test.ts

```diff
@@ -1112,45 +1115,95 @@ describe('POST /referrals/:referralId/service-category/:service-category-id/desi
     },
   ]
 
-  beforeEach(() => {
-    const serviceCategory = serviceCategoryFactory.build({ desiredOutcomes, name: 'social inclusion' })
-    const referral = draftReferralFactory.serviceCategorySelected(serviceCategory.id).build()
+  describe('for a single-service referral', () => {
+    beforeEach(() => {
+      const serviceCategory = serviceCategoryFactory.build({
+        id: 'b33c19d1-7414-4014-b543-e543e59c5b39',
+        desiredOutcomes,
+        name: 'social inclusion',
+      })
+      const referral = draftReferralFactory.serviceCategoriesSelected([serviceCategory.id]).build()
 
-    interventionsService.getDraftReferral.mockResolvedValue(referral)
-    interventionsService.getServiceCategory.mockResolvedValue(serviceCategory)
+      interventionsService.getDraftReferral.mockResolvedValue(referral)
+      interventionsService.getServiceCategory.mockResolvedValue(serviceCategory)
+    })
+
+    it('updates the referral on the backend and redirects to the complexity level page', async () => {
+      await request(app)
+        .post('/referrals/1/service-category/b33c19d1-7414-4014-b543-e543e59c5b39/desired-outcomes')
+        .type('form')
+        .send({ 'desired-outcomes-ids[]': [desiredOutcomes[0].id, desiredOutcomes[1].id] })
+        .expect(302)
+        .expect('Location', '/referrals/1/complexity-level')
```

Comment:
for single service referrals shouldn't this url also be scoped to the service category?